SFUsat/sfu_cmds.h

```diff
@@ -98,18 +98,16 @@
  */
 #define CMD_DATA_MAX_SIZE (14)
 
-typedef enum TASK_IDS {
-	TASK_MAIN,
-	TASK_RADIO,
-	TASK_TICKLE,
+#define TASK_MAIN           (0x00)
+#define TASK_RADIO          (0x01)
+#define TASK_TICKLE         (0x02)
 	/**
 	 * The following tasks are used only in development and not in release.
```

Comment:
Instead of "the following" it's only `TASK_BLINKY` now, so change the comments accordingly.